frontend/src/pages/HomePage.jsx

```diff
@@ -118,6 +118,31 @@ function HomePage() {
     handleModalView();
   }
 
+  /***
+   * Toggles between two feeds.
+   */
+  function toggleFeeds(event) {
+    if (event.target.className.includes("recommended")) {
+      setFeedType(true);
+    } else {
+      setFeedType(false);
+    }
+  }
```

Comment:
You could also just do `setFeedType(event.target.className.includes("recommended"))`